### ISSUE TITLE
HiGHS-v1.2.2 contains files from 1.2.1

### ISSUE THREAD

**Issue by douglaswaynepotter** [JuliaPackaging/Yggdrasil#5725]
Some files from 1.2.1 are in
https://github.com/JuliaBinaryWrappers/HiGHS_jll.jl/releases/tag/HiGHS-v1.2.2%2B0
but only files from 1.2.2 should be.

Steps to confirm:
```
wget https://github.com/JuliaBinaryWrappers/HiGHS_jll.jl/releases/download/HiGHS-v1.2.2%2B0/HiGHS.v1.2.2.x86_64-linux-gnu-cxx11.tar.gz
tar xf HiGHS.v1.2.2.x86_64-linux-gnu-cxx11.tar.gz 
grep  HIGHS_VERSION_PATCH include/HConfig.h 
```
the final command outputs
`#define HIGHS_VERSION_PATCH 1`

But it should be
`#define HIGHS_VERSION_PATCH 2`

**Comment by giordano** [comment]
CC @odow

**Comment by odow** [comment]
That's because the constant wasn't updated before the v1.2.2 release was tagged. It got updated a few days later:

https://github.com/ERGO-Code/HiGHS/commit/993a4e84fd7d858eace4e05182102f2333ffe8b2

It's not something I'm going to go back and fix. Julian just released v1.3.0, so I'm about to build that.

**Comment by giordano** [comment]
I'm going to close the ticket, as far as I understand this was an upstream issue (at least we're bug-compatible with the released package), and a newer version is now also available . Let us know if there is anything else to be done here.
